tests/utils.py

```diff
@@ -43,6 +44,19 @@
 __email__ = '''<cvanoverbeek@schubergphilis.com>'''
 __status__ = '''Development'''  # "Prototype", "Development", "Production".
 
+with open('tests/fixtures/matching_findings.json', encoding='utf-8') as matching_findings_file:
+    findings_fixture = json.load(matching_findings_file)
+    with open('tests/fixtures/non_matching_findings.json', encoding='utf-8') as non_matching_findings_file:
```

Comment:
Why is non-matching-findings loaded twice?

Reply:
The idea is that `findings_fixture` has both matching and non-matching findings.

So when we apply the suppression rules, we are sure that the non-matching findings are ignored.

The other part where we load only non-matching findings was to test that, if there are no matching findings, an empty list is returned.